### ISSUE TITLE
Build error: React.Children.only expected to receive a single React element child.

### ISSUE THREAD

**Issue by jacqui** [news-catalyst/next-tinynewsdemo#190]
After fixing other build and AMP validation errors, my build is now failing with the following message, repeated several times, always referencing `Link` - so at least I think I know where to fix it. 

I have a feeling this is coming from `localiseText()` failing to return a string, maybe? Or perhaps I'm missing a call to localise the text and the component is getting `<someField>.values[]` instead of a string.

```
[ ===] info  - Generating static pages (12/51)
Error occurred prerendering page "/en-US/articles/pandemic/a-culture-of-exploitation-emerges-after-onset-of-coronavirus". Read more: https://err.sh/next.js/prerender-error
Error: Minified React error #143; visit https://reactjs.org/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Object.only (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react/cjs/react.production.min.js:19:241)
    at Link (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/.next/server/pages/tinycms/config.js:868:33)
    at d (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:33:498)
    at bb (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:36:16)
    at a.b.render (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:42:43)
    at a.b.read (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:41:83)
    at exports.renderToString (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:52:138)
    at Object.renderPage (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/next/dist/next-server/server/render.js:50:851)
    at Function.getInitialProps (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/.next/server/pages/_document.js:273:19)
    at loadGetInitialProps (/Users/jacqui/Projects/newscatalyst/review-webiny/next-tinynewsdemo/node_modules/next/dist/next-server/lib/utils.js:5:101)
```

**Comment by jacqui** [comment]
Fixed in PR #186

**Comment by jacqui** [comment]
(This was caused by me putting the global <style> in the <Head> instead of keeping it where it was at the end of the <body> during debugging)
